src/monitor/btcaddrs.go

```diff
@@ -0,0 +1,143 @@
+package monitor
+
+import (
+	"encoding/json"
+	_ "encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+	"github.com/skycoin/teller/src/addrs"
+	"github.com/skycoin/teller/src/util/httputil"
+)
+
+type BtcAddressJSON struct {
+	Addresses []string `json:"btc_addresses"`
+}
+
+type CountBtc struct {
+	All  int
+	Used int
+	Free int
+}
+
+// method: POST
+// url: /addbtc  {"btc_addresses": ["1st_add", "2nd_adr", ...]}
+var AddBtcAddresses = func(w http.ResponseWriter, r *http.Request) {
+	_, err := addrs.NewBTCAddrs(logrus.WithField("prefix", "addrs"), database, r.Body)
+	if err != nil {
+		fmt.Fprintf(w, "failed.")
```

Comment:
Need to send http error codes. See teller/http.go for example

Reply:
done